packages/manager/src/components/ToastNotifications/ToastNotifications.spec.js

```diff
@@ -2,47 +2,85 @@ const { navigateToStory } = require('../../../e2e/utils/storybook');
+  const variants = [
+    'success',
+    'warning',
+    'error',
+    'info'
+  ];
+  const toast = '[data-qa-toast] div';
+  const toastColor = '[data-qa-toast] [role="alertdialog"]'
 
-    it('Toast notification displays with expected variant', () => {
-        variants.forEach((variant) => {
-            notificationButton(variant).click();
-            $(toast).waitForVisible(constants.wait.normal);
-            expect($(toast).getAttribute('class')).toContain(`SnackBar-${variant}`);
-            $(toast).waitForVisible(constants.wait.normal, true);
-        });
-        notificationButton('default').click();
-        $(toast).waitForVisible(constants.wait.normal);
-        expect($(toast).getAttribute('class')).not.toContain('variant');
-        $(toast).waitForVisible(constants.wait.normal, true);
-    });
+  const notificationButton = (state) => {
+    const button = $$('#root>[type="button"]').find(button => button.getText() === state);
+    button.waitForDisplayed(constants.wait.normal);
+    return button;
+  }
 
-    it('Toast notification dissapears after 4 seconds', () => {
-        notificationButton(variants[0]).click();
-        browser.pause(4500);
-        expect($(toast).isVisible()).toBe(false);
-    });
+  const checkToastColor = (variant, color) => {
+    notificationButton(variant).click();
+
+    expect($(toastColor).getCSSProperty('border-left-color').value)
+      .withContext(`incorrect variant color`)
+      .toBe(color);
+    $('[title="Dismiss Notification"]').click()
+    $(toastColor).waitForDisplayed(constants.wait.short, true);
+  }
+
+  beforeAll(() => {
+    navigateToStory(component, childStory);
+  });
 
-    it('No more than 3 notifications display at once', () => {
-        notificationButton(variants[0]).click();
-        notificationButton(variants[1]).click();
-        notificationButton(variants[1]).click();
-        notificationButton(variants[1]).click();
-        browser.pause(500);
-        const successIsNull = $$(toast).find(toast => toast.getAttribute('class').includes('SnackBar-success'));
-        expect(successIsNull).toEqual(undefined);
+  it('Toast notification displays with expected variant', () => {
+    variants.forEach((variant) => {
+      notificationButton(variant).click();
+      $(toast).waitForDisplayed(constants.wait.normal);
+      expect($(toast).getAttribute('class'))
+        .withContext(`incorrect variant type`)
+        .toContain(`SnackBar-${variant}`);
+      $(toast).waitForDisplayed(constants.wait.normal, true);
     });
+
+    notificationButton('default').click();
+    $(toast).waitForDisplayed(constants.wait.normal);
+    expect($(toast).getAttribute('class'))
+      .withContext(`should not contain variant class`)
+      .not.toContain('variant');
+    $(toast).waitForDisplayed(constants.wait.normal, true);
+  });
+
+  it('Toast notifications have correct color', () => {
+    //default
+    checkToastColor('default', 'rgba(0,0,0,0)')
```

Comment:
I wonder if we could import the Theme into this test, like the storybook component does and reference the same values as the component does from the theme, instead of hardcoding these colors:

Relevant code:
https://github.com/linode/manager/blob/develop/packages/manager/src/components/SnackBar/SnackBar.tsx#L5
https://github.com/linode/manager/blob/develop/packages/manager/src/components/SnackBar/SnackBar.tsx#L18